packages/design-system/src/styles/components/_Review.scss

```
@@ -0,0 +1,16 @@
+@import '../settings/index.scss';
+
+.ds-c-review {
+  border-bottom: 2px solid $border-color;
+  display: flex;
+  justify-content: space-between;
+  padding-bottom: 16px;
+  padding-top: 16px;
+}
+
+.ds-c-review__heading {
+  display: inline-block;
+  font-weight: $font-bold;
```

I think this font size needs to be added. Take a look at this local branch and compare to production and you will see the production header is smaller than locally. 

```suggestion
  font-weight: $font-bold;
  font-size: $base-font-size;
```

---

nice catch!